### ISSUE TITLE
Vampire's and Wight King's Yellow Line

### ISSUE THREAD

**Issue by AlarielleRadiant** [chadvandy/cbfm_wh3#1932]
Vampire's yellow line may belong to Wight King, and Wight King's to Vampire.
![image](https://github.com/user-attachments/assets/6690a3e9-9010-44c7-a9d0-169a53af9f70)
![image](https://github.com/user-attachments/assets/4132d79f-5221-44c7-a209-c79fb76b6017)

**Comment by mfvreeland** [comment]
No evidence this is an error.

**Comment by allenhu0912** [comment]
The flavor text for Vampire's Dark Benediction actually says that it's a Wight King, but I think it must have been something that CA changed very early on, just like how Necromancers shout "Purple Sun" when you use them for agent actions even though they don't use Lore of Death.

**Comment by mfvreeland** [comment]
@allenhu0912 
Fair point. It has been like this for as long as I can remember, which makes me think it was a design change, but it clearly wasn't executed perfectly.

**Comment by mfvreeland** [comment]
After discussing this on Discord, it was decided that changing this would be beyond scope.
